gubernator/view_build.py

```diff
@@ -129,16 +129,57 @@ def get(self, prefix, job, build):
             failures=failures, build_log=build_log, pr=pr, pr_digest=pr_digest,
             testgrid_query=testgrid_query))
 
+@view_base.memcache_memoize('build-list://', expires=60)
+def build_list((job_dir, before)):
```

Comment:
It's a weird side-effect of how the memcache_memoize decorator works-- it doesn't support more than one argument. This should be cleaned up in a future refactor.